domains/*.hrv-clan.is-local.org.json

```diff
@@ -0,0 +1,13 @@
+{
+    "description": "Website introducing Discord server: Hrv Rero Clan and more!",
+    "domain": "is-local.org",
+    "subdomain": "*.hrv-clan",
+    "owner": {
+        "repo": "https://github.com/hungrerovn/hrv-clan",
+        "email": "hungrero.vn@gmail.com"
+    },
+    "record": {
+        "CNAME": ["github.com/hungrerovn/hrv-clan"]
```

Comment:
CNAME cannot be in an array. Make sure to also register the root wildcard domain of domains/hrv-clan.is-local.org.json